### DISCUSSION TITLE
Can Microsoft still track me if I am using vscodium?

### DISCUSSION THREAD

**Discussion by Raj2032** [VSCodium/vscodium#1387] [General]
Even though all telemetry is disabled by default, is there anyway microsoft is still collecting data from vscodium?

**Comment by daiyam** [comment]
Yes, if you are using extensions from MS or from devs which are using MS telemetry (directly included in the extension).

**Comment by Raj2032** [comment]
I have created my own theme extension and published it to the ms store. I did not put in any telemetry. When I build it does it insert MS telemetry? Even if I downloaded the extension right from their store and installed it into vscodium would it have MS telemetry?
------- Original Message -------

**Reply by daiyam** [reply]
No

**Comment by Raj2032** [comment]
Ok thanks :)
------- Original Message -------
